README.md

```diff
@@ -18,60 +18,32 @@ The [Amazon Elastic Block Store](https://aws.amazon.com/ebs/) Container Storage
 
 ## Container Images
 
-|Driver Version | [GCR](https://us.gcr.io/k8s-artifacts-prod/provider-aws/aws-ebs-csi-driver ) Image | [ECR](https://gallery.ecr.aws/ebs-csi-driver/aws-ebs-csi-driver) Image |
-|---------------------------|--------------------------------------------------|-----------------------------------------------------------------------------|
-|v1.15.0                    |k8s.gcr.io/provider-aws/aws-ebs-csi-driver:v1.15.0| public.ecr.aws/ebs-csi-driver/aws-ebs-csi-driver:v1.15.0                    |
+| Driver Version | [registry.k8s.io](https://kubernetes.io/blog/2022/11/28/registry-k8s-io-faster-cheaper-ga/) Image | [ECR Public](https://gallery.ecr.aws/ebs-csi-driver/aws-ebs-csi-driver) Image |
```

Comment:
Should we still link to the existing Kubernetes GCR registry where our images are being hosted?

Reply:
We discussed this offline. There is no real equivalent link for `registry.k8s.io` and the blog post is probably the most helpful resource for a user trying to answer the question "what is `registry.k8s.io` and where are my images coming from?"